documentation/asciidoc/topics/proc_deploying_code.adoc

```diff
@@ -0,0 +1,31 @@
+[id='deploying_code-{context}']
+= Deploying Custom Code
+Create a persistent volume and make custom code available on {brandname} clusters.
```

Comment:
Gotcha on the analogy, which is a good one. TBH I'm a little thrown by this because I thought a PVC is more of a request for a chunk of storage. I guess I've got some learning to do, which is always good!

Reply:
Also, @ryanemerson, in this line: https://github.com/infinispan/infinispan-operator/pull/856/files#diff-00eee882033c02da685d0d4b43c5cc2adb79573f712f7b80ec10d0970b798863R9 

Is it not the case that the PVC has to point to a PV?

Reply:
> TBH I'm a little thrown by this because I thought a PVC is more of a request for a chunk of storage.

It is. In the same way a pod is a request for a node's CPU/Mem resources. From the docs "PersistentVolumeClaims are a way for users to "claim" durable storage (such as a GCE PersistentDisk or an iSCSI volume) without knowing the details of the particular cloud environment."

> Is it not the case that the PVC has to point to a PV?

No you don't have to point to a specific PV, but you can do this if you have a PV with specific properties/configuration that you desire. For example, if you wanted to use a NAS PV then it makes sense for the PVC to explicitly reference the required PV.

If you only require a PV of specific type, but don't care about the exact PV that's provisioned you can use `spec.storageClassName` and the cluster will attempt to provision a PV of that type. Similarly, if no `storageClassName` is set, then the cluster default PV type is assumed.

https://kubernetes.io/docs/concepts/storage/persistent-volumes/#lifecycle-of-a-volume-and-claim